plrust/src/lib.rs

```diff
@@ -10,6 +10,27 @@ Use of this source code is governed by the PostgreSQL license that can be found
 #![doc = include_str!("../../README.md")]
 #![forbid(unsafe_op_in_unsafe_fn)]
 
+#[cfg(all(
+    feature = "trusted",
+    not(all(
+        target_os = "linux",
+        any(target_arch = "x86_64", target_arch = "aarch64")
+    ))
+))]
+compile_error!("This platform does not support the 'trusted' version of plrust");
+
+cfg_if::cfg_if! {
+    if #[cfg(feature = "trusted")] {
+        /// We are a trusted language handler.  This will cause plrust user functions to be compiled
+        /// with `postgrestd`
+        pub(crate) const TRUSTED:bool = true;
```

Comment:
extremely minor and tedious nit, nonetheless,
```suggestion
        pub(crate) const TRUSTED: bool = true;
```